README.md

```diff
@@ -30,11 +30,11 @@ Note that C#=>VB or VB=>C# disassembly shouldn't be used to convert between lang
 ### Execution
 
 You can use "Run" target to execute your code and see the output.  
-Execution enables a few nice features such as flow arrows � see here:  
+Execution enables a few nice features such as flow arrows — see here:  

```

Comment:
I wonder what changed here...

Reply:
As far as I can tell, this PR changed encoding of the file from Windows-1252/1250 to UTF-8. GitHub interpreted that as a change on this line, because the encoding of the em-dash changed from 0x97 to UTF-8-encoded U+2014.